docs/wallet-integration-guide/src/exchange-integration/extensions.rst

```diff
@@ -6,16 +6,94 @@ Integration Extensions
-
-  * a common setup is to use two nodes with threshold two; thus both of them must confirm a tx for it to be committed
-
-* upload all .dars for your onboarded tokens on all of these validators
-* use any of the nodes hosting the party to read from as part of history ingestion and to prepare and execute transactions
-
-  * switching between the nodes can be done. take care to re-synchronize the ingestion offset as described in :ref:`restore-from-validator-node-backup`
+As part of setting up the ``treasuryParty``, you need to sign the
+``PartyToParticipant`` mapping which defines the set of confirming
+nodes for that party as well as the confirmation threshold. The
+confirming nodes for the party are essential to keep your party secure
+and compromise of them could lead to loss of funds. Refer to the trust
+model `trust model
+<https://docs.digitalasset.com/overview/3.3/explanations/canton/external-party.html#party-trust-model>`_
+for more details.
+
+To guard against compromise, you can setup your party with multiple
+confirming nodes and a threshold N > 1. As long as less than N nodes
+are compromised, your party is still secured. Common setups are:
+
+1. 2 confirming nodes with a threshold of 2. This provides security
+   against a single node being compromised. Note that it does not
+   provide availability in case of compromise though: If one node is
+   compromised or down, transactions will be rejected as you will not
+   be able to reach 2 valid confirmations. To be able to run
+   transactions again you either need to bring both nodes back online
+   or change the hosting nodes.
+2. 3 confirming nodes with a threshold of 2. This extends the previous
+   setup to also provide availability in case one of the nodes goes
+   down or gets compromised as you can still reach 2 valid
+   confirmations.
+
+Party Setup
+^^^^^^^^^^^
+
+.. TODO:: https://github.com/hyperledger-labs/splice-wallet-kernel/issues/272 Update this when wallet SDK support is available
+
+As part of the :ref:`initial party setup <create-an-external-party>`_,
+you can specify multiple multiple confirming participants and a
+threshold as needed. Note that at this point, the wallet SDK library
+does not yet support this so you must go directly through the Canton
+APIs. This is expected to change soon. Until then, the easiest way to do so at the moment is through the Canton
+console. You can find a full reference for all required steps in the
+`integration test
+<https://github.com/digital-asset/canton/blob/3c9ac9891c03cb06303736d7224bcc01dbd50084/community/app/src/test/scala/com/digitalasset/canton/integration/tests/jsonapi/ExternalPartyLedgerApiOnboardingTest.scala#L183>`_. Note
+in particular that you must sign the ``PartyToParticipant`` mapping
+not just by your party's key but also by all confirming
+participants. This is accomplished through the
+``participant2.topology.transactions.authorize`` step in the test.
+
+DAR Management
```

Comment:
```suggestion
.dar File Management
```
I currently use that abberviation. Feel free to decide and search and replace in either direction.